src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
```

Comment:
If there is no special reason to keep the commented import, please delete it or add some description to the comment in order to explain this.